ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
This project is configured to use sqlite by default for the primary database. In my local secure.py I have switched over to postgresql. Django will not default to using sqlite for the test database automatically as far as I can tell.